HW03/andrey_shynkarou/spec/andrey_shynkarou_hw_03_t_03_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require_relative '../andrey_shynkarou_hw_03_t_03'
+
+RSpec.describe Homework3 do
+  let(:obj) { Object.const_get('Homework3').new }
```

Comment:
we usually use`subject` for such things 
```suggestion
  subject(:obj) { Object.const_get('Homework3').new }
```